{
  "Faults": [
    {
      "latitude": 51.18453319260688,
      "type": "HV",
      "reference": "KX3568",
      "etaUpdate": true,
      "affectedCustomerCount": 63,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "159b35a4-525e-4154-92dd-5e5db5aeafbe",
      "minLongitude": -2.257,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.150623569646521,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-22T17:28:00.000+0000",
      "estimatedRestoration": "2023-05-23T11:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3568'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3568",
      "maxLongitude": -2.1177,
      "minLatitude": 51.0856,
      "name": "BA12 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BA12 0ER",
        "BA12 6BR",
        "BA12 7BA",
        "BA12 7BB",
        "BA12 7BD",
        "BA12 7BG",
        "BA12 7BQ",
        "BA12 9HH",
        "BA12 9HJ",
        "BA12 9HT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.257,
              51.2045
            ],
            [
              -2.1177,
              51.2045
            ],
            [
              -2.1177,
              51.0856
            ],
            [
              -2.257,
              51.0856
            ],
            [
              -2.257,
              51.2045
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2045,
      "etrUpdate": true
    },
    {
      "latitude": 51.37782519456211,
      "type": "LV",
      "reference": "KX3728",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-05-23T00:45:00.000+0000",
      "affectedCustomerCount": 22,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "33f7362b-08b0-4051-93ae-d7c0440f9833",
      "minLongitude": -0.864,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8602971615061324,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-22T22:54:00.000+0000",
      "estimatedRestoration": "2023-05-23T11:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3728'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3728",
      "maxLongitude": -0.8561,
      "minLatitude": 51.3769,
      "name": "RG40 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG40 4EH",
        "RG40 4JE",
        "RG40 4TY",
        "RG40 4TZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.864,
              51.3791
            ],
            [
              -0.8561,
              51.3791
            ],
            [
              -0.8561,
              51.3769
            ],
            [
              -0.864,
              51.3769
            ],
            [
              -0.864,
              51.3791
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-23T00:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3791,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-19T16:12:00.000+0000",
      "latitude": 50.79084708046399,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX1538'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX1538",
      "reference": "KX1538",
      "etaUpdate": true,
      "name": "BH23 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH23 6BG"
      ],
      "location": {
        "coordinates": [
          -1.8597515963189666,
          50.79084708046399
        ],
        "type": "Point"
      },
      "UUID": "18eaf705-0691-4940-a35b-121f632546e3",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -1.8597515963189666,
      "resolved": false
    },
    {
      "latitude": 51.487569701537055,
      "type": "HV",
      "reference": "KX3783",
      "etaUpdate": true,
      "affectedCustomerCount": 512,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "89cecc09-949c-43c6-9cb1-fb6e371204c1",
      "minLongitude": -0.3229,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.2846597629134739,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T05:12:00.000+0000",
      "estimatedRestoration": "2023-05-23T08:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3783'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3783",
      "maxLongitude": -0.2494,
      "minLatitude": 51.4806,
      "name": "TW8 & W4 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "TW8 0EF",
        "TW8 0EX",
        "TW8 0FD",
        "TW8 0FG",
        "TW8 0FJ",
        "TW8 0FS",
        "TW8 0HF",
        "TW8 0HR",
        "TW8 0HS",
        "TW8 0HT",
        "TW8 9DZ",
        "W4 2RU",
        "W4 3NL",
        "W4 3NN",
        "W4 3NP",
        "W4 3NR",
        "W4 3NS",
        "W4 3NT",
        "W4 3NU",
        "W4 3PA",
        "W4 3PF",
        "W4 3PL",
        "W4 3PU",
        "W4 3RJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3229,
              51.4929
            ],
            [
              -0.2494,
              51.4929
            ],
            [
              -0.2494,
              51.4806
            ],
            [
              -0.3229,
              51.4806
            ],
            [
              -0.3229,
              51.4929
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-23T05:50:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "35a2ac00-5e5b-4c31-b6d9-9df22fa13992"
      ],
      "maxLatitude": 51.4929,
      "etrUpdate": true
    },
    {
      "latitude": 51.42741603932591,
      "type": "HV",
      "reference": "KX3809",
      "etaUpdate": true,
      "affectedCustomerCount": 160,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "65beb64a-e8ae-4c78-a842-200166e597ed",
      "minLongitude": -2.1656,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1383476912311608,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T05:35:00.000+0000",
      "estimatedRestoration": "2023-05-23T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3809'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3809",
      "maxLongitude": -2.1185,
      "minLatitude": 51.4089,
      "name": "SN13 & SN15 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "SN13 9QH",
        "SN13 9QJ",
        "SN13 9QQ",
        "SN13 9QW",
        "SN14 0RW",
        "SN15 2JU",
        "SN15 2LZ",
        "SN15 2NA",
        "SN15 2ND",
        "SN15 2NE",
        "SN15 2NF",
        "SN15 2NG",
        "SN15 2NS",
        "SN15 2NU",
        "SN15 2NX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1656,
              51.4479
            ],
            [
              -2.1185,
              51.4479
            ],
            [
              -2.1185,
              51.4089
            ],
            [
              -2.1656,
              51.4089
            ],
            [
              -2.1656,
              51.4479
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-23T06:05:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4479,
      "etrUpdate": true
    },
    {
      "latitude": 50.93349428492712,
      "type": "LV",
      "reference": "KX3808",
      "etaUpdate": true,
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "c0a9b3c6-d82a-4b9a-b2ba-f16c1f73c512",
      "minLongitude": -1.8216,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8202612477816293,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T05:31:00.000+0000",
      "estimatedRestoration": "2023-05-23T09:15:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3808'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3808",
      "maxLongitude": -1.8144,
      "minLatitude": 50.9323,
      "name": "SP6 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SP6 1PA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.8216,
              50.9365
            ],
            [
              -1.8144,
              50.9365
            ],
            [
              -1.8144,
              50.9323
            ],
            [
              -1.8216,
              50.9323
            ],
            [
              -1.8216,
              50.9365
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-23T06:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9365,
      "etrUpdate": true
    },
    {
      "latitude": 50.909128513639835,
      "type": "LV",
      "reference": "KX3723",
      "etaUpdate": true,
      "affectedCustomerCount": 24,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "67870937-bc3a-43eb-bad9-356b841c523f",
      "minLongitude": -1.3137,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3080246627923608,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T05:52:00.000+0000",
      "estimatedRestoration": "2023-05-23T09:25:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3723'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3723",
      "maxLongitude": -1.3042,
      "minLatitude": 50.9049,
      "name": "SO30 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO30 0BH",
        "SO30 0BS",
        "SO30 0BT",
        "SO30 4AB",
        "SO30 4AF",
        "SO30 4AG",
        "SO30 4AJ",
        "SO30 4DG",
        "SO30 4DQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3137,
              50.9128
            ],
            [
              -1.3042,
              50.9128
            ],
            [
              -1.3042,
              50.9049
            ],
            [
              -1.3137,
              50.9049
            ],
            [
              -1.3137,
              50.9128
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9128,
      "etrUpdate": true
    },
    {
      "latitude": 58.20249708053937,
      "type": "HV",
      "reference": "KX3713",
      "etaUpdate": true,
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "c2f35e3a-a078-4837-951a-c734a7c0ca8b",
      "minLongitude": -6.3849,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.382134499473198,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-22T20:00:00.000+0000",
      "estimatedRestoration": "2023-05-23T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3713'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3713",
      "maxLongitude": -6.3778,
      "minLatitude": 58.2015,
      "name": "HS1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HS1 2RS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -6.3849,
              58.2065
            ],
            [
              -6.3778,
              58.2065
            ],
            [
              -6.3778,
              58.2015
            ],
            [
              -6.3849,
              58.2015
            ],
            [
              -6.3849,
              58.2065
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-22T20:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 58.2065,
      "etrUpdate": true
    },
    {
      "latitude": 56.46725697699222,
      "type": "HV",
      "reference": "KX3764",
      "etaUpdate": true,
      "affectedCustomerCount": 5,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "2a6b53a5-fd0c-429f-a6eb-2547d6f43db8",
      "minLongitude": -2.9363,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.931974546134054,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T03:44:00.000+0000",
      "estimatedRestoration": "2023-05-23T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3764'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3764",
      "maxLongitude": -2.9299,
      "minLatitude": 56.465,
      "name": "DD1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD1 3LZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.9363,
              56.4686
            ],
            [
              -2.9299,
              56.4686
            ],
            [
              -2.9299,
              56.465
            ],
            [
              -2.9363,
              56.465
            ],
            [
              -2.9363,
              56.4686
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-23T04:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.4686,
      "etrUpdate": true
    },
    {
      "latitude": 58.92538042387733,
      "type": "PSI",
      "reference": "216282",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "b26d25f2-9afa-4cde-8385-5b611a951f66",
      "minLongitude": -2.928,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.8152638814141855,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T05:00:00.000+0000",
      "estimatedRestoration": "2023-05-31T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '216282'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/216282",
      "maxLongitude": -2.6566,
      "minLatitude": 58.8912,
      "name": "KW17 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "KW17 2QJ",
        "KW17 2SA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.928,
              58.9733
            ],
            [
              -2.6566,
              58.9733
            ],
            [
              -2.6566,
              58.8912
            ],
            [
              -2.928,
              58.8912
            ],
            [
              -2.928,
              58.9733
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 58.9733,
      "etrUpdate": true
    },
    {
      "latitude": 56.18648199505268,
      "type": "LV",
      "reference": "KX3781",
      "etaUpdate": true,
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "88c6cdd4-14b1-47bf-9eaf-347a7e37e8ae",
      "minLongitude": -3.9621,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.959054917904653,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T05:12:00.000+0000",
      "estimatedRestoration": "2023-05-23T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX3781'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX3781",
      "maxLongitude": -3.9541,
      "minLatitude": 56.1832,
      "name": "FK15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "FK15 0DJ",
        "FK15 0DU",
        "FK15 0LJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -3.9621,
              56.1883
            ],
            [
              -3.9541,
              56.1883
            ],
            [
              -3.9541,
              56.1832
            ],
            [
              -3.9621,
              56.1832
            ],
            [
              -3.9621,
              56.1883
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.1883,
      "etrUpdate": true
    },
    {
      "latitude": 57.89094039538854,
      "type": "PSI",
      "reference": "216548",
      "etaUpdate": true,
      "affectedCustomerCount": 112,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "1dd8de3a-1f25-43f9-b54d-f073e206a09f",
      "minLongitude": -5.1554,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.136463466482707,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-23T07:00:00.000+0000",
      "estimatedRestoration": "2023-05-23T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '216548'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/216548",
      "maxLongitude": -5.0766,
      "minLatitude": 57.8781,
      "name": "IV26 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV26 2SW",
        "IV26 2SX",
        "IV26 2SZ",
        "IV26 2TB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -5.1554,
              57.9041
            ],
            [
              -5.0766,
              57.9041
            ],
            [
              -5.0766,
              57.8781
            ],
            [
              -5.1554,
              57.8781
            ],
            [
              -5.1554,
              57.9041
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.9041,
      "etrUpdate": true
    }
  ]
}
